### PR TITLE
Add embedded svg from a configurable source

### PR DESCRIPTION
**Pull request by lildude**
I like my emojis to be the embedded SVG code rather than adding many more calls to an external server.

This PR adds this functionality.

I've also updated the README and added tests for this functionality.

### CODE REVIEW

**Comment by lildude** [comment]
Ooops, this was meant to be a PR against my own fork :blush:.  Sorry for the noise.  You're welcome to take these changes if you want though.

**Comment by omninonsense** [comment]
Oh, on the contrary. I love the idea of embedding the emojis! Also, I am flattered someone even took interest! :grin:

**Comment by lildude** [comment]
:cool: :tada:

File: lib/jekyll/emoji/filter.rb [hunk 1]
```diff
@@ -10,12 +10,13 @@ module Filter
       # @param [String] output_format
       # @param [FalseClass|TrueClass|NilClass] ascii
       # @param [FalseClass|TrueClass|NilClass] shortname
+      # @param [FalseClass|TrueClass|NilClass] src
```
Comment (omninonsense) [inline lib/jekyll/emoji/filter.rb]: You might've meant  `# @param [String|NilClass] src`?

**Comment by lildude** [comment]
If you want to see it in action, the emojis at https://barefootrunner.co.uk/, eg those in https://barefootrunner.co.uk/week-in-review-29-feb-6-mar-16, are now embedded using these changes with `src` in my `_config.yml` pointing to a local copy of the EmojiOne SVG files (I build locally).

File: lib/jekyll/emoji/filter.rb (hunk 1 above)
Comment (lildude) [inline lib/jekyll/emoji/filter.rb]: :blush: Indeed. I blame blindly copying & pasting.

**Comment by omninonsense** [comment]
Looks like it works! If you don't plan on making more changes I can merge it. :wink:

**Comment by lildude** [comment]
Go for it. I don't plan on making anymore changes right now.

**Pull request merged**